pkg/math/vote-weighted/math.go

```diff
@@ -1,4 +1,4 @@
-package math
+package voteweighted
```

Comment:
So it looks like we have a discrepancy between the package name `voteweighted` and the dir name `vote-weighted`. 

I say we just use `voteweighted`.

Also the file `math.go` should probably be `voteweighted.go`